coderd/users.go

```diff
@@ -316,6 +328,18 @@ func (api *API) postUser(rw http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	err = userpassword.Validate(req.Password)
```

Comment:
Also needs to be in the user change password endpoint

Reply:
I think there already is now